pm/sender_test.go

```diff
@@ -91,6 +91,49 @@ func TestSenderEV(t *testing.T) {
 	assert.Zero(ticketEV(ticketParams.FaceValue, ticketParams.WinProb).Cmp(ev))
 }
 
+func TestValidateSender(t *testing.T) {
+	assert := assert.New(t)
+	account := accounts.Account{
+		Address: RandAddress(),
+	}
+	am := &stubSigner{
+		account: account,
+	}
+	rm := &stubRoundsManager{round: big.NewInt(5), blkHash: [32]byte{5}}
+	sm := newStubSenderManager()
+	sm.info[account.Address] = &SenderInfo{
+		Deposit:       big.NewInt(100000),
+		WithdrawRound: big.NewInt(0),
+	}
+	s := &sender{
+		signer:            am,
+		roundsManager:     rm,
+		senderManager:     sm,
+		maxEV:             big.NewRat(100, 1),
+		depositMultiplier: 2,
+	}
+	// GetSenderInfo error
+	sm.err = errors.New("GetSenderInfo error")
+	err := s.validateSender()
+	assert.EqualError(err, "unable to validate sender: could not get sender info: GetSenderInfo error")
+	sm.err = nil
+
+	// Sender's (withdraw round + 1 <= current round)
+	sm.info[account.Address].WithdrawRound = big.NewInt(4)
```

Comment:
Test for when withdraw round + 1 = current round? (Missed this in the initial review)

Reply:
the current round is set to `5`, so would `4 + 1 ` fulfil that scenario ?  We don't have a test testing `withdrawRound + 1 < current round` though.

Reply:
Ah yes my mistake. We are missing a test for when withdraw round + 1 < current round.

Reply:
added test acefc0d